src/Utils/BlockScaleGasPriceFetchingService.ts

```diff
@@ -0,0 +1,24 @@
+import fetch from 'node-fetch';
+import { IGasPriceFetchingService, GasPriceEstimation } from '../Types';
+import BigNumber from 'bignumber.js';
+
+export class BlockScaleGasPriceFetchingService implements IGasPriceFetchingService {
+  private apiAddress = 'https://dev.blockscale.net/api/gasexpress.json';
```

Comment:
So why was ethgasstation removed?

Reply:
Agree, I'm not sure about BlockScale availability so it was a good idea to have ethgasstation as failsafe implementation.

Reply:
IMO we should use ethgasstation as primary since it usually is what people refer to when making transactions and setting the gas price themselves. It should be more accurate for the reason that more people are using it as a reference. We can then use blockscale as the backup.

Reply:
MyCrypto is using BlockScale so people scheduling tx may essentially use it as suggestion. On the other hand the small differences between ethgasstation and blockscale are not relevant.

Reply:
> We can also get more information from the ethgasstation API like the estimated wait times which I don't see on the blockscale response. We will need these data points for the modifications to the execution actions in the timenode.

Yes, but we also know the time data from blockscale, it would be hardcoded though. But what we really need is information like:

my tx is gonna be in 1-2 blocks for 90% if I use X amount of GWei. And both services provides that as FAST/FASTEST.